app/controllers/hyrax/downloads_controller.rb

```diff
@@ -39,7 +92,8 @@ def derivative_download_options
     end
 
     def file_set_parent(file_set_id)
-      file_set = Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: file_set_id, use_valkyrie: Hyrax.config.use_valkyrie?)
```

Comment:
the version of [#find_by_alternate_identifier](https://github.com/samvera/valkyrie/blob/v3.0.3/lib/valkyrie/persistence/postgres/query_service.rb) that exists in valkyrie `v3.0.3` does not account for a `use_valkyrie` param, so this method call erred.

we decided to define `use_valkyrie` in "lib/wings/valkyrie/query_service" instead since the valkyrie defined method should always expect `use_valkyrie` to be true.